teachertool/src/state/helpers.ts

```diff
@@ -43,10 +44,24 @@ export function verifyRubricIntegrity(rubric: Rubric): {
     return { valid: invalidCriteria.length === 0, validCriteria, invalidCriteria };
 }
 
-export function isProjectLoaded(state: AppState) {
+export function isProjectLoaded(state: AppState): boolean {
     return !!state.projectMetadata;
 }
 
+export function isRubricLoaded(state: AppState): boolean {
+    return !!(state.rubric.criteria.length || state.rubric.name);
+}
+
+export function getSafeProjectName(state: AppState): string | undefined {
```

Comment:
Is the "safe" addition to the function name just because even if we don't have anything in state, we'll serve the type that's needed back?

Reply:
The addition of "safe" is to indicate the caller will get back a value that is always friendly to display. For untitled projects, it returns "Untitled Project" so that a falsey check on the result will never pass when a project is loaded.

Reply:
Makes sense. So this function returning `undefined` works in the case where there is no project loaded, though, because in that case, nothing would be displayed anyway.